tests/FSharp.Data.Tests/HtmlParser.fs

```diff
@@ -805,15 +805,16 @@ let ``Can parse pre blocks``() =
     result |> should equal [ "\r\n        This code should be indented and\r\n        have line feeds in it" ]
 
 [<Test>]
-let ``Can parse code blocks``() =
-    let html = "<code>\r\n        let f a b = a * b\r\n        f 5 6 |> should equal 30</code>"
+let ``Can parse pre blocks with char refs``() =
```

Comment:
This removes the code blocks test - could you add that test back as well?

Reply:
Sure, it would test for invalid behavior though since code is an inline element. Should I restore the behavior for code anyways and open another issue for elements nested in pre?